internal/informer/appproject/projectinformer.go

```diff
@@ -179,5 +185,24 @@ func NewAppProjectInformer(ctx context.Context, client appclientset.Interface, o
 			return pi.filterFunc(o)
 		}),
 	)
-	return i, applisters.NewAppProjectLister(i.Indexer()), err
+	pi.projectInformer = i
+	pi.projectLister = applisters.NewAppProjectLister(i.Indexer())
+	return pi, err
+}
+
+func (i *AppProjectInformer) Start(ctx context.Context) {
+	log().Infof("Starting app project informer (namespaces: %s)", strings.Join(i.namespaces, ","))
+	err := i.projectInformer.Start(ctx)
+	if err != nil {
+		log().Errorf("Failed to start app project informer: %v", err)
+		return
+	}
+	log().Infof("project informer has shutdown")
```

Comment:
This log statement is misleading. The project informer keeps running in the background at this point in time.